Dockerfile

```diff
@@ -13,13 +13,8 @@ RUN ARCH=$(uname -m) ; echo $ARCH \
 
 FROM base AS executor-img
 
-RUN ARCH=$(uname -m) ; echo $ARCH ; \
-        if [[ $ARCH == "ppc64le" ]] ; then \
-        echo "Downloading https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/36.20220906.3.1/ppc64le/fedora-coreos-36.20220906.3.1-qemu.ppc64le.qcow2.xz" && \
-        curl -s -o coreos_production_qemu_image.qcow2.xz https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/36.20220906.3.1/ppc64le/fedora-coreos-36.20220906.3.1-qemu.ppc64le.qcow2.xz && \
-        unxz coreos_production_qemu_image.qcow2.xz ; \
-        else \
-        echo $ARCH && \
+RUN if [[ -z "$arg" ]] ; then \
```

Comment:
what is "$arg"?

Reply:
I just reverted back to the original code. I feel it was added before to improve readability. 
Let me remove it.